src/index.js

```diff
@@ -221,6 +231,62 @@ exports.createFromPrivKey = function (key, callback) {
   })
 }
 
+exports.createFromProtobuf = function (buf, callback) {
```

Comment:
Could you just somehow use .createFromJSON so that we don't have the same logic twice?

Reply:
.createFromJSON expects the data to be in base64. I could make a common function out of the validation logic.

Reply:
@mkg20001 that would be good.